src/openzaak/management/commands/setup_configuration.py

```diff
@@ -80,6 +84,22 @@ def handle(self, *args, **options):
             notif_config.save()
 
             # Step 2
+            # This service should be created in migration
+            #   0002_move_config_to_service_model
+            service = Service.objects.get(label="Notificaties API")
+            service.api_root = notifications_api_root
+            service.auth_type = AuthTypes.zgw
+            random_client_id = (
+                f'open-zaak-{"".join(random.choices(string.ascii_letters, k=10))}'
+            )
```

Comment:
rather use `from django.utils.crypto import get_random_string` instead for this kind of stuff. It essentially does the same, but I prefer using framework functions.